app/src/main/java/com/example/drowsydriverapp/MainActivity.kt

```
@@ -161,5 +197,73 @@ fun MainContent(
+                text = "Frame: $frameText | Mem: $memText",
+                style = MaterialTheme.typography.bodyMedium
+            )
+            Text(
+                text = "Battery: $batteryText @ $tempText",
+                style = MaterialTheme.typography.bodyMedium
+            )
+        }
+    }
+}
+
+private fun Float.format(): String = if (this == 0f) "0" else String.format("%.1f", this)
```

detekt (1.23.8)

[warning] String.format("%.1f", this) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)